core/chains/evm/log/registrations.go

```diff
@@ -410,6 +410,10 @@ func (r *handler) sendLog(log types.Log, latestHead evmtypes.Head,
 		if len(filters) > 0 && len(log.Topics) > 1 {
 			topicValues := log.Topics[1:]
 			if !filtersContainValues(topicValues, filters) {
+				logger.Debugw("filters did not contain expected topic",
+					"blockNumber", log.BlockNumber, "blockHash", log.BlockHash,
+					"address", log.Address, "latestBlockNumber", latestBlockNumber,
+					"topicValues", topicValues, "filters", filters)
```

Comment:
What is the type of `filters` here? Is it `[]common.Hash`?

Reply:
If it's `[][]byte` then this log won't be very digestable, we'd need to hex-ify it.

Reply:
yeah filters is `[][]Topic`, where Topic is `common.Hash`. Updated the code to hexify it.